tests/test_cell_metadata.py

```diff
@@ -0,0 +1,33 @@
+import sys
+import unittest
+
+sys.path.append("../ingest")
+from cell_metadata import CellMetadata
+
+
+class TestCellMetadata(unittest.TestCase):
+    def test_validate_header_for_coordinate_values_false(self):
+        """Ensures validate_header_for_coordinate_values returns false when
+        coordintate value is in metadata file
```

Comment:
```suggestion
        coordinate value is in metadata file
```

And similarly throughout this file.

Reply:
0f2b689